### PR TITLE
Open Conversation API

### PR DESCRIPTION
**Pull request by spolu**
r? @philipperolet

### CODE REVIEW

File: front/pages/api/v1/w/[wId]/assistant/[cId]/messages/[mId]/events.ts [hunk 1]
```diff
@@ -16,23 +16,12 @@ async function handler(
...
-    });
-  }
-
   const { auth, keyWorkspaceId } = await Authenticator.fromKey(
     keyRes.value,
     req.query.wId as string
   );
 
-  if (keyWorkspaceId !== req.query.wId) {
+  if (!auth.isBuilder() || keyWorkspaceId !== req.query.wId) {
```
Comment (philipperolet) [inline front/pages/api/v1/w/[wId]/assistant/[cId]/messages/[mId]/events.ts:24]: nit: Add " for builders" in the error message

Comment (philipperolet) [reply front/pages/api/v1/w/[wId]/assistant/[cId]/messages/[mId]/events.ts:24]: (5 occurences)

**Review by philipperolet — APPROVED** [review]
LGTM :fire: small nit

File: front/pages/api/v1/w/[wId]/assistant/[cId]/messages/[mId]/events.ts (hunk 1 above)
Comment (spolu) [reply front/pages/api/v1/w/[wId]/assistant/[cId]/messages/[mId]/events.ts:24]: A good api key is necessarily a builder one

**Pull request merged**